app/src/main/java/org/wikipedia/feed/aggregated/AggregatedFeedContentClient.kt

```diff
@@ -140,13 +140,13 @@ class AggregatedFeedContentClient {
         private fun requestAggregated() {
             aggregatedClient.clientJob?.cancel()
             val date = DateUtil.getUtcRequestDateFor(age)
-            aggregatedClient.clientJob = CoroutineScope(Dispatchers.Default).launch(
+            aggregatedClient.clientJob = CoroutineScope(Dispatchers.Main).launch(
                 CoroutineExceptionHandler { _, caught ->
                     L.v(caught)
                     cb.error(caught)
                 }
             ) {
-                withContext(Dispatchers.Main) {
+                withContext(Dispatchers.IO) {
```

Comment:
This whole `withContext... ` brace is actually unnecessary. We're already inside a coroutine scope, and network calls will be automatically placed in a `IO` context.

Reply:
Done!